api/nomics.py

```diff
@@ -0,0 +1,125 @@
+import datetime
+import calendar
+import connexion
+
+from services.bitshares_websocket_client import client as bitshares_ws_client
+from services.bitshares_elasticsearch_client import client as bitshares_es_client
+from services.cache import cache
+import api.explorer
+import es_wrapper
+import config
+
+
+def info():
+    return {
+        "name": "Bitshares",
+        "description": "An exchange description of at least 1000 characters in plain text (no html)",
+        "location": "Anywhere",
+        "logo": "https://example.com/exchange-logo.png",
+        "website": "https://bitshares.org/",
+        "twitter": "example",
+        "capability": {
+            "markets": True,
```

Comment:
[Api](https://github.com/nomics-crypto/nomics-platform/blob/master/doc/cryptocurrency-api-exchange-integration.md#info---exchange-information---required) expects **t**rue / **f**alse (lowercase)

Reply:
response is lowercase, please check http://185.208.208.184:5000/nomics/info

Reply:
right my bad sorry.